MonumentAddons.cs

```diff
@@ -5197,6 +5197,14 @@ public override void ObjectSpawned(SpawnPointInstance instance)
                     SpawnedVehicleComponent.AddToVehicle(Adapter.PluginInstance, instance.gameObject);
                     entity.Invoke(() => DisableVehicleDecay(entity), 5);
                 }
+
+                var hackableCrate = entity as HackableLockedCrate;
+                if ((object)hackableCrate != null && hackableCrate.shouldDecay)
```

Comment:
Currently, this will only enable decay after hack for the `codelockedhackablecrate` prefab, not for the `codelockedhackablecrate_oilrig` prefab. One of my goals with #36 was to improve consistency between the prefabs, so that users can discover either prefab and not have to read documentation to understand which to use, as they would function basically the same.

If we share the goal of aligning the prefabs, we should also enable decay after hack for the `codelockedhackablecrate_oilrig` prefab. That can be achieved by scheduling the `SetHackableCrateDecay` invoke regardless of the value of `shouldDecay`.

Reply:
Oh I need to think more when Im doing something. Original issue was to disable decay on `codelockedhackablecrate ` so I automatically ignored `codelockedhackablecrate_oilrig` . Good catch

Reply:
Honestly, I think you're doing a great job of thinking of things. Not only does it come through in your comments and PRs that you are thinking of many technical edge cases, but you are also thinking about how to make the plugin intuitive and solve various end-user problems that weren't originally defined. I'm very impressed!